src/events/SDL_mouse.c

```diff
@@ -557,6 +558,11 @@ int SDL_SendMouseMotion(Uint64 timestamp, SDL_Window *window, SDL_MouseID mouseI
     return SDL_PrivateSendMouseMotion(timestamp, window, mouseID, relative, x, y);
 }
 
+int SDL_SendMouseRawMotion(Uint64 timestamp, SDL_MouseID mouseID, Sint32 dx, Sint32 dy)
+{
+    return SDL_PrivateSendMouseRawMotion(timestamp, mouseID, dx, dy);
```

Comment:
You don't need a private implementation, you could just inline this function here.